docs/index.rst

```diff
@@ -19,3 +21,43 @@ The configuration of the *Kafka* plugin is very simple, consisting of four param
   - **Send JSON**: This controls how JSON data points should be sent to Kafka. These may be sent as strings or as JSON objects.
 
   - **Data Source**: Which Fledge data to send to Kafka; Readings or Fledge Statistics.
+
++-----------+
+| |kafka_2| |
++-----------+
+
+  - **Security Protocol**: Security protocol to be used to connect to the kafka broker.
+
+  - **Mechanism**: Authentication mechanism to be used to connect to kafka broker. Only PLAIN mechanism is supported as of now, support for other authentication mechanism will be added soon
```

Comment:
I suggest to keep it as SASL Mechanism; this is what Kafka and Azure event hub speaks; and a user configuring the plugin must be able to map things (per terminology) easily.

Reply:
Fixed 
https://en.m.wikipedia.org/wiki/Simple_Authentication_and_Security_Layer